src/js/components/Layer.js

```diff
@@ -35,16 +35,18 @@ class LayerContents extends Component {
       firstFocusable.focus();
     }
 
+    this._keyboardHandlers = {
+      tab: this._processTab
+    };
+
     if (this.props.onClose) {
-      this._keyboardHandlers = {
-        tab: this._processTab,
-        esc: this.props.onClose
-      };
-      KeyboardAccelerators.startListeningToKeyboard(
-        this, this._keyboardHandlers
-      );
+      this._keyboardHandlers['esc'] = this.props.onClose;
```

Comment:
i believe we can write this as `this._keyboardHandlers.esc`. That's why the building is failing!